### ISSUE TITLE
File export broken after a filename containing '#' is processed

### ISSUE THREAD

**Issue by Sesu8642** [thestonehead/ThunderbirdAttachmentExtractor#6]
After doing an export, I noticed that many attachments were not actually saved to disk. I figured out that everything worked fine until a specific file containing a '#' in the original filename was processed. The target filename does not seem to matter. I determined the messenger.saveAllAttachments function to cause the problem. After it encounters a problematic file, it stops and does not continue with the next file. This might be a bug in the saveAllAttachments function or possibly incorrect usage. Unfortunately, I am unable to find proper documentation on this function call.

**Comment by thestonehead** [comment]
Yeah, I had to wrangle that one out of Thunderbird source code. I'll try and look into it.
